spec/ics-2-consensus-verification/README.md

```diff
@@ -0,0 +1,190 @@
+---
+
+ics: 2
+title: Consensus Verification
+stage: proposal
+category: ibc-core
+requires: 23
+required-by: 3
+author: Juwoon Yun <joon@tendermint.com>, Christopher Goes <cwgoes@tendermint.com>
+created: 2019-02-25
+modified: 2019-04-01
+
+---
+
+## Synopsis
+
+This standard specifies the properties that consensus algorithms of chains implementing IBC are 
+expected to satisfy. The properties are needed for efficient and safe verification in the higher
+level protocol abstractions. The algorithm who uses these properties to verify substates of 
+another chain is referred to as a "light client".
+
+## Specification
+
+### Motivation
+
+`Header` defines required properties of the blockchain on the network. The implementors can 
```

Comment:
This paragraph is very confusing. How `Header` defines the properties of the blockchain?